recipe/gen_patch_json.py

```diff
@@ -2504,7 +2504,7 @@ def _gen_new_index_per_key(repodata, subdir, index_key):
             record["version"] in {"1.2.0", "1.3.0", "1.3.1"} and
             record.get("timestamp", 0) < 1678096271000
         ):
-            _replace_pin("conda >=4.8", "conda >=4.14", record["depends"], record)
+            _replace_pin("conda >=4.8,<23.4", "conda >=4.14<23.4", record["depends"], record)
```

Comment:
This needs a comma in the second arg.

Reply:
Fixed (can't believe you commented in the minute the typo was in).